samples/boot/oauth2resourceserver/README.adoc

```diff
@@ -0,0 +1,161 @@
+and each makes a query to the Resource Server with their corresponding token.
+
+The Resource Server subsquently verifies with the Authorization Server and authorizes the request, returning the phrase
+
+```bash
+Hello, {subject}!
+```
+
+where "subject" is the value of the `sub` field in the JWT returned by the Authorization Server.
+
+== 2. Running the app
+
+To run as a stand-alone application, do:
+
+```bash
+./gradlew bootRun
+```
+
+Or import the project into your IDE and run `OAuth2ResourceServerApplication` from there.
+
+Once it is up, you can retreive a valid JWT token from the authorization server, and then hit the endpoint:
+
+```bash
+curl -H "Authorization: Bearer $TOKEN" localhost:8081
+```
+
+Which will respond with the phrase:
+
+```bash
+Hello, {subject}!
+```
+
+where `subject` is the value of the `sub` field in the JWT returned by the Authorization Server.
+
+=== How do I obtain a valid JWT token?
+
+Getting a valid JWT token from an Authorization Server will vary, depending on your setup. However, it will typically
+look something like this:
+
+```bash
+curl --user $CLIENT_ID:$CLIENT_PWD -d "grant_type=client_credentials&scope=message:read" $AUTH_ENDPOINT/token
+```
+
+which will respond with a JSON payload containing the `access_token` among other things:
+
+```json
+{
+  "access_token" : "{the access token}",
+  "token_type" : "Bearer",
+  "expires_in" : "{an expiry}",
+  "scope" : "{a list of scopes}"
+}
+```
+
+For example, the following can be used to hit the mock Authorization Server for a valid JWT token:
+
+```bash
+curl --user id:password -d "grant_type=client_credentials&scope=message:read" http://localhost:8082/token
+```
+
+Which will give a response similar to this (formatting mine):
+
+```json
+{
+  "access_token": "eyJhbGciOiJSUzI1NiJ9.eyJzY29wZSI6Im1lc3NhZ2U6cmVhZCIsImV4cCI6MzEwNjMyODEzMSwianRpIjoiOGY5ZjFiYzItOWVlMi00NTJkLThhMGEtODg3YmE4YmViYjYzIn0.CM_KulSsIrNXW1x6NFeN5VwKQiIW-LIAScJzakRFDox8Ql7o4WOb0ubY3CjWYnglwqYzBvH9McCFqVrUtzdfODY5tyEEJSxWndIGExOi2osrwRPsY3AGzNa23GMfC9I03BFP1IFCq4ZfL-L6yVcIjLke-rA40UG-r-oA7r-N_zsLc5poO7Azf29IQgQF0GSRp4AKQprYHF5Q-Nz9XkILMDz9CwPQ9cbdLCC9smvaGmEAjMUr-C1QgM-_ulb42gWtRDLorW_eArg8g-fmIP0_w82eNWCBjLTy-WaDMACnDVrrUVsUMCqx6jS6h8_uejKly2NFuhyueIHZTTySqCZoTA",
+  "token_type": "Bearer",
+  "expires_in": 1576800000,
+  "scope": "message:read"
+}
+```
+
+Then, using that access token:
+
+```bash
+curl -H  "Authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJzY29wZSI6Im1lc3NhZ2U6cmVhZCIsImV4cCI6MzEwNjMyODEzMSwianRpIjoiOGY5ZjFiYzItOWVlMi00NTJkLThhMGEtODg3YmE4YmViYjYzIn0.CM_KulSsIrNXW1x6NFeN5VwKQiIW-LIAScJzakRFDox8Ql7o4WOb0ubY3CjWYnglwqYzBvH9McCFqVrUtzdfODY5tyEEJSxWndIGExOi2osrwRPsY3AGzNa23GMfC9I03BFP1IFCq4ZfL-L6yVcIjLke-rA40UG-r-oA7r-N_zsLc5poO7Azf29IQgQF0GSRp4AKQprYHF5Q-Nz9XkILMDz9CwPQ9cbdLCC9smvaGmEAjMUr-C1QgM-_ulb42gWtRDLorW_eArg8g-fmIP0_w82eNWCBjLTy-WaDMACnDVrrUVsUMCqx6jS6h8_uejKly2NFuhyueIHZTTySqCZoTA" \
+  localhost:8081
+```
+
+I get:
```

Comment:
Avoid the use of "I" in documentation as it is not inclusive of the reader. Generally, I find it good to avoid personal pronouns, but if you do use them leverage ones that are inclusive of the reader (i.e. we)